manifests/profiles/README.profiles

```diff
@@ -0,0 +1,15 @@
+Kubernetes Vendor Optimized Profiles
+
+Vendor optimized profiles are approved as per this RFC:
+https://docs.google.com/document/d/1N_yC9vi4Jt3auMLLSOkavQb6qgbJKr-Mp4kUL3GOmEc/edit#heading=h.xw1gqgyqs5b
+
+If you wish to add a Kubernetes optimized profile, please join the Environments
+WG meeting with your two lead maintainers for the profile. The working group
+ will provide further instructions. Please read the RFC.
+
++----------------------+---------------+-----------------+------------------+
+| K8s Platform         | Vedor         | Lead Maintainer | Lead Maintainer  |
```

Comment:
typo Vendor

Reply:
Plain text. I can convert to MD if you prefer - I think it would be more useful. @ostromart  thoughts?

Reply:
+1 on markdown

Reply:
converted to markdown.